fedmsg/crypto/utils.py

```diff
@@ -132,18 +132,12 @@ def _load_remote_cert(location, cache, cache_expiry, tries=0, **config):
         (cache_expiry and time.time() - modtime > cache_expiry)
     ):
         try:
-            response = requests.get(location)
+            with requests.Session() as session:
+                session.mount('http://', requests.adapters.HTTPAdapter(max_retries=tries))
+                session.mount('https://', requests.adapters.HTTPAdapter(max_retries=tries))
+                response = session.get(location, timeout=30)
             with open(cache, 'w') as f:
-                f.write(response.content)
-        except requests.exceptions.ConnectionError:
-            if tries < 3:
```

Comment:
It does seem like the default number of tries is being changed in this patch, from 3 to 1? Maybe we should default tries to 3 now?